lib/hooks/useRoleCapabilities/useRoleCapabilities.js

```diff
@@ -15,7 +15,16 @@ import {
 import { CAPABILITIES_LIMIT } from '../../constants';
 import { getCapabilitiesGroupedByTypeAndResource } from '../../utils';
 
-export const useRoleCapabilities = (roleId, tenant = '', expand = false, options = {}) => {
+/**
+ * Get capabilities for a given role.
+ * 
+ * @param {string} roleId The Role ID.
+ * @param {string} tenant The Tenant ID. Defaults to `stripes.okapi.tenant`.
```

Comment:
This statement is false. The previous implementation had a default value of empty string, but now there is no default value. `useOkapiKy` will use `stripes.okapi.tenant` if the value it receives is falsy. 

Given the only call to this hook is from RoleDetailsCapabilitiesAccordion, and you already updated that call, is it necessary to remove the default value?

Reply:
I can leave the default value since `''` is falsey. I will update the comment to indicate that `useOkapiKy` will default to `stripes.okapi.tenant` since that is very helpful to be aware of.